src/core/public/plugins/plugins_service.test.ts

```diff
@@ -64,6 +64,10 @@ function createManifest(
     requiredPlugins: required,
     optionalPlugins: optional,
     requiredBundles: [],
+    owner: {
+      name: 'Core',
+      githubTeam: '@elastic/kibana-core',
```

Comment:
Why is this one `@elastic/kibana-core` and other ones are `elastic/kibana-core` and other ones are `kibana-core`?  Doesn't seem like we're consistent.

Reply:
It should just be `kibana-core`, I'll try and fix the places I got this wrong and will update the documentation so it's clearer.